cucumber/api/features/support/rest_helpers.rb

```diff
@@ -49,6 +52,32 @@ def get_json(path, options = {})
     set_result result
   end
 
+  def get_json_no_default_headers(path, options = {})
```

Comment:
We should a comment above this function explaining the reason we're using `Net::HTTP` instead of `RestClient`.  Something like:

```
# Since there is no way to remove the default Accept header in RestClient
# we use Net::HTTP here.  Otherwise, we wouldn't be able to simulate requests
# that omit the Accept header in our tests.
```